### DISCUSSION TITLE
Managing pg_service.conf via QDT?

### DISCUSSION THREAD

**Discussion by afourneyron** [Guts/qgis-deployment-cli#482] [Q&A]
Hello l'équipe ! 😃 
A question about the pg_service.conf, is it possible to manage the file via the scenario?
And if we can go further with two different profiles and scenarios with different pg_service.conf, how does the management work if there are both scenarios deployed on the same workstation ?

Thanks for all your hard work !

**Comment by Guts** [comment]
Hello @afourneyron,

QDT is a tool for deploying QGIS-related customizations, not for deploying anything to do with any QGIS-related software. Even if it's PostgreSQL since pg_service.conf is used for every PostgreSQL client on the workstation: QGIS, ArcGIS, pgAdmin, DBeaver, DicoGIS... whatever!

Still, a QDT job which would be able to copy and fill files using environment variables or keyring entries could be a great asset for the project. But not so easy to do.
